zkevm-circuits/src/tx_circuit.rs

```diff
@@ -117,18 +116,9 @@ pub(crate) fn tx_to_sign_data(tx: &Transaction, chain_id: u64) -> Result<SignDat
             e
         })?;
     // msg = rlp([nonce, gasPrice, gas, to, value, data, sig_v, r, s])
-    let mut stream = RlpStream::new_list(9);
-    stream
-        .append(&tx.nonce)
-        .append(&tx.gas_price)
-        .append(&tx.gas_limit)
-        .append(&tx.to.unwrap_or_else(Address::zero))
-        .append(&tx.value)
-        .append(&tx.call_data.0)
-        .append(&chain_id)
-        .append(&0u32)
-        .append(&0u32);
-    let msg = stream.out();
+    let geth_tx: geth_types::Transaction =
```

Comment:
I think this conversion is redundant (it's converting from `geth_types::Transaction` to `geth_types::Transaction`)

Reply:
Good catch! Thanks! 
All these names are really confusing!!!

Reply:
Sure! Good catch. All these names are confusing.
Fixed in 6ce4c1d